carma/launch/plugins.launch.py

```diff
@@ -394,13 +354,10 @@ def generate_launch_description():
                 ],
                 parameters=[ platoon_tactical_ihp_param_file, vehicle_config_param_file ]
             ),
-            
         ]
     )
 
-    
-
     return LaunchDescription([    
         carma_plugins_container,
-        platooning_plugins_container
+        #platooning_plugins_container
```

Comment:
platooning_plugins_container should not be deleted and needs to be added back. Not just this line. Looks like the whole container was removed from the launch

Reply:
on develop somehow platooning container was duplicated, so I removed one of them